CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ option(USE_ETCD "option for enable etcd as metadata server" ON)
 option(USE_ETCD_LEGACY "option for enable etcd based on etcd-cpp-api-v3" OFF)
 option(USE_REDIS "option for enable redis as metadata server" OFF)
 option(USE_HTTP "option for enable http as metadata server" ON)
+option(USE_LRU_MASTER "option for using LRU in master service" OFF)
+set(LRU_MAX_CAPACITY 1000)
```

Comment:
we'd better configure it at run time, similar to `enable_gc`
https://github.com/kvcache-ai/Mooncake/blob/14af70c9cadffa91824ca43331839e78d07050a4/mooncake-store/src/master.cpp#L14